### PR TITLE
Add full transaction log retrieval and summary generation 

### PR DESCRIPTION
**Pull request by zmoore-usgs**

### CODE REVIEW

**Comment by zmoore-usgs** [comment]
The big drop in coverage is because the areas of added code are covered by integration tests that can't run since we can't pull the MLR Legacy DB image anymore due to Artifactory being down.

File: src/main/resources/monitoringLocation.xml [hunk 1]
```diff
@@ -334,18 +350,126 @@
            </set>
          where agency_cd = #{agencyCode, jdbcType=CHAR} and
                site_no = #{siteNumber, jdbcType=CHAR}
-
     </update>
-    
-    <select id="find" resultMap="loggedActionResult">
+
+    <sql id="transactionLogs">
+        SELECT
```
Comment (kmschoep-usgs) [inline src/main/resources/monitoringLocation.xml:356]: I created logged_actions_vw, so the mybatis didn't have to do all the hstore parsing.

File: src/main/java/gov/usgs/wma/mlrlegacy/model/LoggedTransaction.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
+package gov.usgs.wma.mlrlegacy.model;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class LoggedTransaction {
```
Comment (kmschoep-usgs) [inline src/main/java/gov/usgs/wma/mlrlegacy/model/LoggedTransaction.java:10]: couldn't the loggedAction model just be used?

File: src/main/java/gov/usgs/wma/mlrlegacy/dao/LoggedActionsDao.java [hunk 3]
```diff
@@ -17,8 +19,15 @@ public LoggedActionsDao(SqlSession sqlSession) {
 		this.sqlSession = sqlSession;
 	}
 
-	public List<LoggedAction> find(Map<String, Object> queryParams) {
-		return sqlSession.selectList("find", queryParams);
+	public List<LoggedAction> findActions(Map<String, Object> queryParams) {
+		return sqlSession.selectList("findActions", queryParams);
 	}
 
+	public List<LoggedTransaction> findTransactions(Map<String, Object> queryParams) {
```
Comment (kmschoep-usgs) [inline src/main/java/gov/usgs/wma/mlrlegacy/dao/LoggedActionsDao.java:26]: I'm not seeing much difference between findActions and findTransactions.  You could modify findActions to better fit the use case (it was basically a first draft until we knew better what the use cases were).

**Pull request merged**